draft-ietf-quic-qlog-quic-events.md

```diff
@@ -1922,8 +1928,7 @@ ConnectionCloseFrame = {
     ? error_code_value: uint64
     ? reason: text
 
-    ; For known frame types, the appropriate "frame_type" string
-    ; For unknown frame types, the hex encoded frame identifier value
+    ; when error_space === "transport"
```

Comment:
if?

Reply:
Oh I missed this, sorry. 

We seem to use of mix of "if" and "when" throught the docs. We should do a consistency pass as followup.